circuitpython/CIRC02/8_leds.py

```diff
@@ -1,34 +1,28 @@
-# CIRC02 - LED Fun
-# (CircuitPython)
-# this circuit was designed for use with the Metro Express Explorers Guide on Learn.Adafruit.com
+"""
+'8_leds.py'.
 
-# by Limor Fried/Ladyada and Asher Lieber for Adafruit Industries
-# modified by microwattbott
-
-# import required libraries
-import digitalio
-import pulseio
-import board
+=================================================
+lights up 8 leds with different animations
+"""
 import time
+import board
+import digitalio
 
-# create a list of DigitalInOut objects using pins 2-9
-ledpins = [ digitalio.DigitalInOut(board.D2),
-            digitalio.DigitalInOut(board.D3),
-            digitalio.DigitalInOut(board.D4),
-            digitalio.DigitalInOut(board.D5),
-            digitalio.DigitalInOut(board.D6),
-            digitalio.DigitalInOut(board.D7),
-            digitalio.DigitalInOut(board.D8),
-            digitalio.DigitalInOut(board.D9) ]
+ledpins = [digitalio.DigitalInOut(board.D2),
```

Comment:
If staying consistent with pythonic naming, constants should be uppercase underscored: `LED_PINS`